source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/Cim/ValidationErrors/CimValidationErrorTextToken.cs

```diff
@@ -34,5 +34,7 @@ public enum CimValidationErrorTextToken
         DocumentSenderProvidedChargeId,
         DocumentType,
         MeteringPointId,
+        TriggeredBySenderProvidedId,
```

Comment:
`SenderProvidedId` leads the mind to `SenderProvidedChargeId`, so I suggest
```suggestion
        TriggeredByOperationId,
```

Reply:
fixed